services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.tsx

```diff
@@ -101,6 +101,8 @@ export const RateDetails = ({
             : showFileUploadError && !hasValidFiles
             ? ' You must remove all documents with error messages before continuing'
             : undefined
+    const documentsErrorKey =
+        fileItems.length === 0 ? 'rateDocuments' : '#file-items-list'
```

Comment:
oof seeing this I wish we had clearer ids `rate-documents-file-input`.  suspect that change could break tests but I can circle back here to look at this at the end of the sprint.

Reply:
@haworku We might want to pass the list id in as a prop to `FileUpload`, that way the form could dictate the value.